low-level/framework/rabbitmq/rabbitmq_egress_processor.py

```diff
@@ -299,6 +299,9 @@ def _transmit_msg_on_exchange(self):
                 except connection_exceptions:
                     logger.error("RabbitMQegressProcessor, _transmit_msg_on_exchange, rabbitmq connectivity lost, adding message to consul %s" % self._jsonMsg)
                     store_queue.put(jsonMsg)
+                except Exception as err:
+                    logger.error("RabbitMQegressProcessor, _transmit_msg_on_exchange, some problem while publishing the message, adding message to consul %s" % self._jsonMsg)
```

Comment:
keep log msg short
Unknown error while publishing the message, adding to persistent store

Reply:
Ok.